Bottom: 89e7c448fd9b52425517520a882670ef72945c5d
Top:    82f7d0ecef8f4f7be962682956ac6c0ed2c25688
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 14:08:45 +0100

Refresh of p3.patch

---

diff --git a/doc/pcb.texi b/doc/pcb.texi
index bee33c3..bd1744d 100644
--- a/doc/pcb.texi
+++ b/doc/pcb.texi
@@ -2475,7 +2475,7 @@ layout of a circuit board.  There are two independent autorouter
 implementations: Autorouter and experimental (and optional)
 Toporouter.
 
-The autorouter is a rectangle-expansion type of autorouter based on 
+The autorouter is a rectangle-expansion type of autorouter based on
 ``A Method for Gridless Routing of Printed Circuit Boards'' by
 A. C. Finch, K. J. Mackenzie, G. J. Balsdon, and G. Symonds in the
 1985 Proceedings of the 22nd ACM/IEEE Design Automation Conference.
@@ -2546,7 +2546,7 @@ optimizer.  See section @ref{Trace Optimizer} for more information on
 the trace optimizer.
 
 
-The autorouter is a rectangle-expansion type of autorouter based on 
+The autorouter is a rectangle-expansion type of autorouter based on
 ``A Method for Gridless Routing of Printed Circuit Boards'' by
 A. C. Finch, K. J. Mackenzie, G. J. Balsdon, and G. Symonds in the
 1985 Proceedings of the 22nd ACM/IEEE Design Automation Conference.
